appengine-standard-java8/springboot-appengine-standard/pom.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.google.appengine.demos</groupId>
+	<artifactId>springboot-appengine-standard</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>war</packaging>
+
+	<name>springboot-appengine-standard</name>
+	<description>Demo project for Spring Boot</description>
+
+	<parent>
```

Comment:
changed the parent. but.. would've loved to keep it as a regular spring boot app.